frontend/app/src/pages/main/managed-workers/$managed-worker/components/update-form.tsx

```diff
@@ -635,16 +639,16 @@ export default function UpdateWorkerForm({
                   )}
                   <Label>Scaling Method</Label>
                   <Tabs
-                    defaultValue="static"
-                    value={hasAutoscaling ? 'autoscaling' : 'static'}
+                    defaultValue="Static"
+                    value={scalingType}
                     onValueChange={(value) => {
-                      if (value === 'static') {
-                        setHasAutoscaling(false);
+                      if (value === 'Static') {
+                        setScalingType('Static');
```

Comment:
extreme nit: can factor this out now that `value` is one of those two, so can just be `setScalingType(value)` and then the if/else logic 👍